src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertyLines = sourceString
+    .split(';')
+    .map(item => item.trim());
+
+  const styles = propertyLines.reduce((result, line) => {
+    const [key, value] = line
+      .split(':')
+      .map(prop => prop.trim());
+
+    if (key && value) {
+      result[key] = value;
+    }
+
+    return result;
+  }, {});
+
+  return styles;
```

Comment:
I think that it is not necessary to create a variable for the value that is immediately returned.

```suggestion
  return  styles = propertyLines.reduce((result, line) => {
    const [key, value] = line.split(':')

    if (key && value) {
      result[key.trim()] = value.trim();
    }

    return result;
  }, {});
```